types/qs/index.d.ts

```diff
@@ -16,7 +16,7 @@ declare namespace QueryString {
         filter?: Array<string | number> | ((prefix: string, value: any) => any) | undefined;
         arrayFormat?: "indices" | "brackets" | "repeat" | "comma" | undefined;
         indices?: boolean | undefined;
-        sort?: ((a: any, b: any) => number) | undefined;
+        sort?: ((a: string, b: string) => number) | undefined;
```

Comment:
This isn’t correct; sort comparators can take any value (including in the language).
```suggestion
        sort?: ((a: unknown, b: unknown) => number) | undefined;
```